frontend/src/static/css/style.scss

```diff
@@ -316,12 +316,12 @@ header{
         }
 
         .contribution{
-            margin-top:0.1rem;
             color:mui-color("red");
+            font-size: 0.875rem;
```

Comment:
Seems unnecessary. But it is OK to leave it here.

Reply:
Yep it is redundant, will remove this line. Thank you.